airbyte-cdk/bulk/toolkits/load-object-storage/src/main/kotlin/io/airbyte/cdk/load/file/object_storage/ObjectStorageFormattingWriter.kt

```diff
@@ -133,17 +142,26 @@ class ParquetFormattingWriter(
     formatConfig: ParquetFormatConfiguration,
     private val rootLevelFlattening: Boolean,
 ) : ObjectStorageFormattingWriter {
+    private val log = KotlinLogging.logger {}
+
     private val pipeline = ParquetMapperPipelineFactory().create(stream)
-    private val avroSchema =
-        pipeline.finalSchema.withAirbyteMeta(rootLevelFlattening).toAvroSchema(stream.descriptor)
-    private val writer =
-        outputStream.toParquetWriter(avroSchema, formatConfig.parquetWriterConfiguration)
+    private val mappedSchema: ObjectType = pipeline.finalSchema.withAirbyteMeta(rootLevelFlattening)
+    private val avroSchema: Schema
+    private val writer: ParquetWriter
+
+    init {
+        log.info { "Preprocessed schema for avro conversion: $mappedSchema" }
```

Comment:
Could this generate very big logs?

Reply:
I think this could go away but the avro schema should still be logged.

Reply:
fixed